app/controllers/users_controller.rb

```diff
@@ -96,8 +96,7 @@ def ensure_enrollment_success
   def make_enrollment_edits
     return unless enroll_params[:role].to_i == CoursesUsers::Roles::STUDENT_ROLE
     # for students only, posts templates to userpage and sandbox
-    WikiCourseEdits.new(action: :enroll_in_course, course: @course,
-                        current_user: current_user, enrolling_user: @user)
+    EnrollmentEditsWorker.edit_enrollment(@course.id, current_user.id, @user.id)
```

Comment:
This is the same worker that just got created as `EnrollInCourseWorker`. Use that new worker instead of adding a new one.